consumer/hikey/hikey960/build/android-mainline.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Build the android-mainline kernel for Hikey960
+permalink: /documentation/consumer/hikey/hikey960/build/android-mainline.md.html
+redirect_from:
+- /documentation/ConsumerEdition/HiKey960/BuildSource/android-mainline.md.html
+- /documentation/consumer/hikey960/build/android-mainline.md.html
```

Comment:
Hi, @pcolmer

Could you please check the links above if there is any problem there, or if all of them are necessary?
I have no idea about the rule here:(
I want to add a link here https://www.96boards.org/documentation/consumer/hikey/hikey960/build/.

Thanks,
Yongqin Liu